### PR TITLE
Chore/update components

### PR DESCRIPTION
**Pull request by GoOz**
- [x] Use new table styles from UI
- [x] Use Nav styles from UI on Drive
- [x] Use `<Nav />` component on Photos (not possible on Drive due to a way too complex logic)

~~⚠️ Needs a new release of Cozy-UI or it will break.~~

### CODE REVIEW

**Comment by y-lohse** [comment]
Probably unrelated to this issue, but the uplaod button on photos has a double icon
capture d ecran 2018-01-18 10 58 46

**Comment by y-lohse** [comment]
Also this : 
capture d ecran 2018-01-18 18 03 02

**Comment by GoOz** [comment]
@y-lohse Well, I'll take a look because the bugs about the nav (didn't look the other one) are not supposed to exist, that's the purpose of this PR. 
Just to be sure, are you sure your loaded Cozy-UI version is 7.1.0?

**Comment by enguerran** [comment]
You should rebase this PR, then I'll run it to see if @y-lohse remarks are still extant.

**Comment by GoOz** [comment]
@y-lohse I can confirm the first one, my fault, the second one too but it's not related to this PR and the last one I don't reproduce.
Working on fixing the first two…

**Comment by GoOz** [comment]
Ok fixed those two. Needs a UI update though. Let you know when it's released.

**Comment by GoOz** [comment]
@y-lohse everythibg should be good now.

File: package.json [hunk 1]
```diff
@@ -137,7 +137,8 @@
     "classnames": "^2.2.0",
     "cozy-bar": "4.7.0",
     "cozy-client-js": "0.6.0",
-    "cozy-ui": "7.0.2",
+    "cozy-client-js": "0.5.0",
```
Comment (y-lohse) [inline package.json]: That line shouldn't be there ;)

Comment (GoOz) [reply package.json]: ah fucking rebase! >_<

**Comment by GoOz** [comment]
Done! I may squash this one…

**Comment by y-lohse** [comment]
Yah I think a squash is the right move 😛

**Comment by enguerran** [comment]
You could squash except the git comment must be less generic than _« Chore/update components »_.

**Pull request merged**